tests-manual/repro_issues_42.R

```diff
@@ -0,0 +1,41 @@
+library(shiny)
```

Comment:
I don't see any output for the plot or text output in this one:

![image](https://cloud.githubusercontent.com/assets/86978/12572612/40f80ecc-c3b4-11e5-9cd5-7633e26a731b.png)